lib/c_docs/doc_member.rb

```diff
@@ -95,7 +95,12 @@ def process_data(node, mapping, platform)
 
     def process_typedef(node, mapping, platform)
       process_return_type(node, mapping, platform)
-      @data[platform]['argsstring'] = node.at_css('argsstring').content.to_s
+      argsstring = node.at_css('argsstring')
```

Comment:
Are there issues with this path as is? I haven't run into errors in this path in my local testing, but it's entirely possible I missed something.